app/models/user.rb

```diff
@@ -103,9 +103,9 @@ class User < ApplicationRecord
   has_many :offer_assignments, through: :pseuds
   has_many :pinch_hit_assignments, through: :pseuds
   has_many :request_claims, class_name: "ChallengeClaim", foreign_key: "claiming_user_id", inverse_of: :claiming_user
-  has_many :gifts, -> { where(rejected: false) }, through: :pseuds
+  has_many :gifts, -> { where(rejected: false) }, through: :pseuds, dependent: :destroy
```

Comment:
Is this still required if we already have `has_many :pseuds, dependent: :destroy` on users?

Reply:
I don't think so. I mostly left it to reinforce that's what happens, but I don't feel strongly about it so I can remove if you'd prefer

Reply:
Let's remove it unless it's necessary, yeah.